ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ethernet-interfaces.md

```diff
@@ -657,31 +657,43 @@ interface Ethernet26.1
    description TENANT_A pseudowire 1 interface
    encapsulation vlan
       client unmatched
+   encapsulation vlan
```

Comment:
This kind of repeated config will be generated if it is defined with `type: l2dot1q or l3dot1q`